node/node-aquilon.inc.php

```diff
@@ -11,32 +11,32 @@ function detail($NODE, $SHORT) {
     $info = False;
     $url = $CONFIG['URL']['AQUILON'] . ":6901/host/".urlencode($NODE);
 
-    $h = get_headers($url);
-    $h = substr($h[0], 9, 3);
-    if ($h == 200) {
+    $header = get_headers($url);
+    $header = substr($header[0], 9, 3);

```

Comment:
This extracts the HTTP status code from the header, so `$status_code` might be better?